src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +45,36 @@
      * Андрей - 600
      * София - 900</p>
      */
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        LocalDate fromDate = LocalDate.parse(dateFrom, formatter);
+        LocalDate toDate = LocalDate.parse(dateTo, formatter);
```

Comment:
try to create method which will parse Strings to Locale date , compare them and return boolean

Reply:
done